usage.txt

```diff
@@ -10,8 +10,10 @@ Options:
   --icon=<path>        Path to the icon file that will be the app icon in the DMG window.
   --icon-size=<px>     How big to make the icon for the app in the DMG. [Default: `80`].
   --background=<path>  Path to a PNG image to use as the background of the DMG.
+  --background-color=<color>  Background color (accepts css colors).
   --format=<format>    Disk image format. [Default: `UDZO`].
   --debug              Enable debug messages.
   --overwrite          Overwrite any existing DMG.
+  --window="<object>"  Window size and position options. ex: "{size: {width: 100, height: 100}, position" {x: 0, y: 0}}"
```

Comment:
That's not really a great way of handling nested params in a CLI interface. I believe minimist has support for options like `--window.size.width=100`.

Reply:
Actually, that documentation is plain wrong there. I overlooked you were using minimist.